drizzlepac/haputils/analyze.py

```diff
@@ -529,28 +534,51 @@ def verify_guiding(filename, min_length=33):
     data -= bkg_limit
     imgarr = np.clip(data, 0, data.max())
 
+    # Build up a mask of all bad pixels and ignore them when looking for
+    # sources and linear features
+    dqarr = None
+    for extn in hdu:
+        if 'extname' in extn.header and extn.header['extname'] == 'DQ':
+            dqarr = hdu[("DQ", 1)].data.copy()
+            break
```

Comment:
Maybe it does not matter in this specific case, but could the hdu[("DQ",1)].data.copy() just be extn.data.copy() to be more generic?  There may only be "1" DQ in the file, but the source code does not necessarily know this.  Just a question really.

Reply:
Good point.  That change was made in 476518e.